about/yocto-wpe.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Try WPE images for Raspberry Pi"
+tags: [about]
+data: { dateless: "true" }
+permalink: /about/yocto-wpe.html
+---
+
+One way to easily test WPE is to use Yocto images for Raspberry Pi
+devices, write them in a micro-SD card and use them to boot the
+device.
+
+
+## Getting and flashing the image on a micro-SD card
+
+### Downloads
+
+#### Download for Raspberry Pi 4 (64-bit)
+
+The current version of WPE in these images is **2.38.x**.
+
+Download image with:
+```
+wget https://wk-contrib.igalia.com/yocto/meta-webkit/browsers/stable/images/raspberrypi4-64-mesa/core-image-weston-browsers-raspberrypi4-64-mesa_202205_4.0_kirkstone_r1_20221228073347.wic.bz2
+```
+
+#### Download for Raspberry Pi 3B/3B+ (32-bit)
+
+The current version of WPE in these images is **2.38.x**.
+
+Download image with:
+```
+wget https://wk-contrib.igalia.com/yocto/meta-webkit/browsers/stable/images/raspberrypi3-mesa/core-image-weston-browsers-raspberrypi3-mesa_202205_4.0_kirkstone_r1_20221228073347.wic.bz2
+```
+
+### Flash the image
+
+To write with '`dd`':
```

Comment:
Can we add a note like _" (replace `mmcblk_XYZ` with your actual target device)"_, right before the colon?